frontend/wecluck-frontend/src/index.css

```diff
@@ -0,0 +1,68 @@
+:root {
```

Comment:
aswell as styled componenets you can read about mui afterwards but there is alot of work anyway